specificatie/klanten/openapi.yaml

```diff
@@ -1046,8 +1084,16 @@ components:
           type: string
           maxLength: 3
           example: "nld"
-        adres:
-          $ref: '#/components/schemas/KlantAdres'
+        correspondentieadres:
+          oneOf:
+            - $ref: '#/components/schemas/Binnenlandsadres'
+            - $ref: '#/components/schemas/Buitenlandsadres'
+            - $ref: '#/components/schemas/Postadres'
+        locatieadres:
```

Comment:
...of bezoekadres? Dat laatste is misschien duidelijker qua onderscheid met correspondentieadres.

Reply:
Ik heb het nu even hernoemd naar `verblijfsadres`.

Reply:
Ik bedoel `verblijfadres` naar aanleiding suggestie Robert